packages/yuudachi/src/commands/utility/sponsor.ts

```diff
@@ -1,12 +1,10 @@
-import { type CommandInteraction, type Snowflake, ButtonStyle, ComponentType } from 'discord.js';
```

Comment:
None of these changes are relevant to this PR.

Reply:
Agreed, reverted in [8781e25](https://github.com/Naval-Base/yuudachi/pull/1069/commits/8781e25a498f6169c2d121d1887139dbc28b0a64)